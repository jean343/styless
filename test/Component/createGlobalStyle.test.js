import React from 'react';
import renderer from 'react-test-renderer';
import {createGlobalStyle} from 'styled-components';
import 'jest-styled-components';

export const getCSS = scope => Array.from(scope.querySelectorAll('style')).map(tag => tag.innerHTML).join('\n');

test('Should theme the global style', () => {
    const GlobalStyle = createGlobalStyle`
    html {
      color: @color;
    }
`;
    renderer.create(<GlobalStyle color="color"/>);
    expect(getCSS(document)).toMatchSnapshot();
});