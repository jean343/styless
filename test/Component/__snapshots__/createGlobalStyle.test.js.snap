// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should theme the global style 1`] = `
"
/* sc-component-id: sc-global-2534925725 */
html{color:color;}"
`;
